src/qml/NightstandPage.qml

```diff
@@ -110,11 +116,37 @@ Item {
                 }
             }
 
+            Item {
+                width: parent.width
+                height: Dims.l(20)
+
+                MouseArea {
+                    anchors.fill: parent
+                    hoverEnabled: true
+                    onClicked: nightstandUseCustomWatchface.value = !nightstandUseCustomWatchface.value
+                    Rectangle {
+                        anchors.fill: parent
+                        color: "white"
+                        opacity: parent.containsPress ? 0.2 : 0
+                    }
+                }
+
+                LabeledSwitch {
+                    anchors.fill: parent
+                    //% "Custom watchface"
+                    text: qsTrId("id-nightstand-custom-watchface")
+                    checked: nightstandUseCustomWatchface.value
+                    opacity: nightstandEnabled.value ? 1.0 : 0.4
+                    onCheckedChanged: nightstandUseCustomWatchface.value = checked
+                }
+            }
+
             Item {
                 width: parent.width
                 height: Dims.l(20)
                 opacity: nightstandEnabled.value ? 1.0 : 0.4
                 enabled: nightstandEnabled.value
+                visible: nightstandUseCustomWatchface.value
```

Comment:
Mh doesn't it mean that even though the item is invisible it still takes vertical space ?
Since all of this is embed in a Flickable, it seems that this would cause a scrollable area for no reason no ?

Reply:
I've tested this as it was a concern of mine as well and it does not take any vertical space when the `Item` is invisible.

Reply:
Alright then :)